collector/rest-servlet/src/main/java/org/apache/karaf/decanter/collector/rest/servlet/RestServletCollector.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.karaf.decanter.collector.rest;
+package org.apache.karaf.decanter.collector.rest.servlet;
```

Comment:
If you change the package here, you have to change the `Private-Package` instruction in the `pom.xml`.

Reply:
You're right, I missed it. Thanks!
Changes done.